### DISCUSSION TITLE
Support for beets v2

### DISCUSSION THREAD

**Discussion by antonmosich** [gtronset/beets-filetote#160] [General]
Are you planning on or even already working on supporting the newest beets release? It seems that most of the tests pass even with beets v2, only those who rely on the `modify_items` function are broken because the function definition changed in  beetbox/beets@3587067c1fed01cc4de4549b33efa2c86ba8453d.

**Comment by gtronset** [comment]
It's very much planned! I've had a handful of personal issues arise since the beginning of the year that have stalled some things, but I'm excited to ensure full support for the latest beets release!

**Reply by antonmosich** [reply]
That's nice to know. Take your time!

**Comment by ChilledSlim** [comment]
Any updates on this?   With the r latest update it ran but didn't do anything with the latest 2.x release

**Reply by gtronset** [reply]
I've been on hiatus much longer than expected due to personal life, but I plan to work on v2 here over the next two weeks. I'll post back once it's ready!

**Comment by gtronset** [comment]
@antonmosich @ChilledSlim [Filetote v1.0.0 has now been released](https://github.com/gtronset/beets-filetote/releases/tag/v1.0.0) and should fully support Beets v2+!
